server.go

```diff
@@ -352,6 +363,18 @@ func WithoutLogger() func(*Server) {
 	}
 }
 
+// WithListener configures the server to use a custom listener.
+// If no listener is provided, it creates a default listener using the server's address.
```

Comment:
```suggestion
// WithListener configures the server to use a custom listener.
```

The sentence is no longer valid, as it  raises a panic

Reply:
sentence removed